rewrite-java/src/main/java/org/openrewrite/java/format/TabsAndIndentsVisitor.java

```diff
@@ -376,16 +376,38 @@ private Space indentTo(Space space, int column) {
         } else {
             space = space.withWhitespace(space.getWhitespace().replaceAll("\t", ""));
         }
-        int indent = findIndent(space);
-        if (!space.getComments().isEmpty()) {
-            indent = findIndent(Space.format(space.getComments().get(space.getComments().size() - 1).getSuffix()));
-        }
 
-        if (indent != column) {
-            int shift = column - indent;
-            space = space.withComments(ListUtils.map(space.getComments(), c ->
-                    indentComment(c, shift)));
-            space = space.withWhitespace(indent(space.getWhitespace(), shift));
+        if (space.getComments().isEmpty()) {
+            int indent = getLengthOfWhitespace(space.getWhitespace());
+            if (indent != column) {
+                int shift = column - indent;
+                space = space.withWhitespace(indent(space.getWhitespace(), shift));
+            }
+        } else {
+            // Note: Formatting multiple block comments on a single line is still not handled.
```

Comment:
Just highlighting the note here.
It should be pretty simple to add a condition to check for Comment.Style, but I'd like to get feedback on what's currently in place.
I plan to add in small chucks, since there are things that need attention in TabsAndIndentVisitor.